guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -0,0 +1,5 @@
+We'd like to thank the following people who contributed to the Foreman {ProjectVersion} release:
+
+Adam Ruzicka, Adi Abramovich, Amir Fefer, Amit Upadhye, Andrew Teixeira, Anna Vitova, Antoine Beaupré, Bastian Schmidt, Ben Magistro, Bernhard Suttner, Chris Roberts, Christopher Smith, Daniel Henninger, Eric D. Helms, Erik Berg, Evgeni Golov, Ewoud Kohl van Wijngaarden, Gordon Bleux, Ian Ballou, Jason, Jeremy Lenz, John Mitsch, Jonathon Turel, Justin Sherrill, Leos Stejskal, Lukáš Zapletal, Marcel Kühlhorn, Marek Hulán, Maria Agaphontzev, Markus Bucher, Melanie Corr, Nadja Heitmann, Nagoor Shaik, Oleh Fedorenko, Ondřej Ezr, Ondřej Pražák, Patrick Creech, Peter Koprda, Rahul Bajaj, Romuald Conty, Ron Lavi, Samir Jha, Sayan Das, Shimon Shtein, Shira Maximov, Tim Meusel, Tomer Brisker, William Clark, Yifat Makias
+
+As well as all users who helped test releases, report bugs and provide feedback on the project.
```

Comment:
Personally, I think this should be a new sentence.
```suggestion
We'd also like to thank all users who helped to test releases, reported bugs and provided feedback on the project.
```

Reply:
I'll be honest and say I just copied it from the official manual and can trace it back to at least https://github.com/theforeman/theforeman.org/commit/d7f7cd592911d107b5144425ce6d731ebfc016ca.

Reply:
I see. I still think it would be an improvement but I guess it can be left as-is for the sake of consistency. Your choice.

Reply:
I'll do that in a later round as part of https://github.com/theforeman/foreman-documentation/issues/1767.